src/frontend/web_application/src/scenes/Timeline/components/MessageItem/presenter.jsx

```diff
@@ -151,7 +168,7 @@ class MessageItem extends Component {
             {this.renderDate()}
           </div>
           <div className="s-message-item__col-select">
-            <input type="checkbox" onChange={this.onCheckboxChange} />
+            <input type="checkbox" onChange={this.onCheckboxChange} id={id} />
```

Comment:
its value is missing for example I select from avatar in small view then I resize the window, the selection should be kept.

Reply:
Can you check last commit https://github.com/CaliOpen/Caliopen/pull/764/commits/8e98188b71510187185d098c5ab7decf118712d2 
I fixed this by passing `isMessageSelected` prop from parent `Timeline`  instead of managing it via `isSelected` state in `MessageItem`

Reply:
👍